components_page.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/css/components_page.css" />
+    <link rel="icon" type="image/x-icon" href="/assests/favicon.ico" />
+    <title>Components | Astro UI</title>
+  </head>
+
+  <body>
+    <div class="page__container">
+      <header class="header">
+        <a href="index.html">
+          <div class="header__logo-container">
+            <img
+              src="/assests/zodiaclogo.png"
+              alt="AtroUI logo"
+              class="logo__img"
+            />
+            <h2 class="header__logo">Astro UI</h2>
+          </div>
+        </a>
+        <nav class="header_nav">
+          <ul>
+            <li><a href="#">Documentation</a></li>
+            <li><a href="https://github.com/ark2002/astro-ui">Github</a></li>
+          </ul>
+        </nav>
+      </header>
+      <main class="main-content">
```

Comment:
You could try to maintain a consistent naming convention. Shouldn't it be `main__content`?

Reply:
> You could try to maintain a consistent naming convention. Shouldn't it be `main__content`?

hey! Did the suggested changes , you can check it out now.

Reply:
Great! 👌